Polyhedron/demo/Polyhedron/Plugins/PMP/Isotropic_remeshing_plugin.cpp

```diff
@@ -407,7 +407,7 @@ public Q_SLOTS:
                  selection_item->constrained_edges_pmap(),
                  get(CGAL::vertex_point, *selection_item->polyhedron()),
                  CGAL::Constant_property_map<face_descriptor, std::size_t>(1),
-                 4. / 3. * target_length))
+                 CGAL::Uniform_sizing_field<FaceGraph>(target_length, pmesh)))
```

Comment:
@sloriot did you remove the 4/3 fraction on purpose?

Reply:
I thought that is was hidden already in the sizing field.